Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3891,8 +3892,13 @@ private ObservableList<ElementInfo> GetAllElementsFromPage(string path, List<eEl
 
             return returnTuple;
         }
-
+        
         ElementInfo IWindowExplorer.LearnElementInfoDetails(ElementInfo EI)
+        {
+            return AddLearnedElementInfoDetails(EI);
```

Comment:
for what to add another function which doing the same, please remove "AddLearnedElementInfoDetails()" and use existing "LearnElementInfoDetails()" instead

Reply:
changed as suggested